variables.tf

```
@@ -141,3 +141,37 @@ variable "tags" {
   description = "A mapping of tags to assign to the PostgreSQL server."
   default     = {}
 }
+
+
+variable "prefix_address" {
+  type        = list(string)
+  description = "the address of the private endpoint"
+  default     = [""]
+}
+
+variable "address_space" {
+  type        = list(string)
+  description = "The addresses space that is used the virtual network."
+  default     = [""]
+}
+variable "subnet" {
+  description = "The subnet object, if you wish to reuse one."
+  type = object({
+    id = string
+  })
+  default = null
+}
+variable "virtual_network" {
+  description = "The virtual_network id, if you wish to reuse one."
+  type = object({
+    id = string
+  })
+  default = null
+}
+
+
+variable "enablePrivateEndpoint" {
```

Le naming de la variable ne respecte pas les standards
```suggestion
variable "enable_private_endpoint" {
```